x-pack/filebeat/input/cel/input.go

```diff
@@ -100,7 +105,7 @@ func (i input) now() time.Time {
 func (input) Name() string { return inputName }
 
 func (input) Test(src inputcursor.Source, _ v2.TestContext) error {
-	cfg := src.(*source).cfg
+	cfg := src.(*source).cfg //nolint:errcheck
```

Comment:
This should be something like `//nolint:errcheck // If this assertion fails, the program is incorrect and should panic.`; golangci-lint requires an explanatory note and the syntax for that is defined, requiring a ` // ` between the linter directive and the reason. Also below.